diff -ur tomboy-0.10.0/configure.in tomboy-0.10.0-addins/configure.in
--- tomboy-0.10.0/configure.in	2008-03-10 15:53:15.000000000 +0100
+++ tomboy-0.10.0-addins/configure.in	2008-03-10 17:32:36.000000000 +0100
@@ -126,9 +126,9 @@
 #
 # Check for external Mono.Addins
 #
-PKG_CHECK_MODULES(MONO_ADDINS, mono-addins > 0.3 \
-	mono-addins-gui > 0.3 \
-	mono-addins-setup > 0.3,
+PKG_CHECK_MODULES(MONO_ADDINS, mono-addins >= 0.3 \
+	mono-addins-gui >= 0.3 \
+	mono-addins-setup >= 0.3,
 	have_external_mono_addins="yes", have_external_mono_addins="no")
 
 if test "x$have_external_mono_addins" != "xyes"; then
