diff -urN tomboy-0.3.3.orig/data/tomboy.desktop.in tomboy-0.3.3/data/tomboy.desktop.in
--- tomboy-0.3.3.orig/data/tomboy.desktop.in	2005-08-23 09:34:25.546703880 +0200
+++ tomboy-0.3.3/data/tomboy.desktop.in	2005-08-23 09:36:34.600084784 +0200
@@ -1,6 +1,5 @@
-
 [Desktop Entry]
-Version=@VERSION@
+Version=0.3.3
 Encoding=UTF-8
 _Name=Tomboy Notes
 _Comment=Quick and handy note-taker
@@ -10,8 +9,7 @@
 StartupNotify=false
 Terminal=false
 Type=Application
-Categories=Application;Utility;
-OnlyShowIn=KDE;
+Categories=GTK;GNOME;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=tomboy
 X-GNOME-Bugzilla-Component=General
