--- tomboy-1.8.3/configure.in.orig	2011-11-14 18:32:07.000000000 +0100
+++ tomboy-1.8.3/configure.in	2022-04-25 22:52:41.045856668 +0200
@@ -172,7 +172,7 @@ AC_SUBST(DBUS_SERVICE_DIR)
 #
 # Check for dbus-sharp
 #
-PKG_CHECK_MODULES(DBUS, dbus-sharp-1.0 >= 0.4 dbus-sharp-glib-1.0 >= 0.3)
+PKG_CHECK_MODULES(DBUS, dbus-sharp-2.0 >= 0.8 dbus-sharp-glib-2.0 >= 0.6)
 AC_SUBST(DBUS_LIBS)
 
 #
--- tomboy-1.8.3/Tomboy/RemoteControlProxy.cs.orig	2011-06-21 02:55:00.000000000 +0200
+++ tomboy-1.8.3/Tomboy/RemoteControlProxy.cs	2022-04-26 06:23:29.941836511 +0200
@@ -52,7 +52,7 @@ namespace Tomboy
 			BusG.Init ();
 
 			RemoteControl remote_control = new RemoteControl (manager);
-			Bus.Session.Register (Namespace,
+			Bus.Session.Register (
 			                      new ObjectPath (Path),
 			                      remote_control);
 
